src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
```

Comment:
The size of the feature doesn't seem prohibitive to me, but it'd be nice to put it in its own file to keep the code complexity in check